src/colorella/colormap.py

```diff
@@ -18,209 +18,226 @@
 
 Colorella Package: Color organizing and easy to learn laboratory
 The packages allows to load color maps from several sources including:
-- matplotlib default color maps (e.g viridis, plasma)
+- matplotlib default colormaps (e.g viridis, plasma)
+- colorcet colormaps (e.g fire, rainbow)
 - cpt files
 - ct files
 - json
 - or to create new colormaps from dictionaries or lists of colors
 classmethods allow the following functions:
-- save a newly or modified colormap as .cpt file
-- turn a colormap to greyscale (e.g. for printing in greyscale
+- save a newly or modified colormap as .cpt, .ct or .json file
+- turn a colormap to greyscale (e.g. for printing in greyscale)
 - reverse a colormap
 - view a colormap as plot
+- load or convert to gdal colortable objects
 - create a list or dictionary object contaning all the colors from a colormap
-- adapt a colormao using a function?
-- adapt the indices of a colormap
+
 '''''
 import os
+import json
 import numpy as np
 import matplotlib as mpl
 import matplotlib.pyplot as plt
 import matplotlib.cm as cm
 import matplotlib.colors as col
-from osgeo import gdal
+import colorcet as cc
 import warnings
+from colorella.conversions import cptfile2dict, ctfile2dict, json2list
 
-from src.colorella.utils import cptfile2dict, gdal2dict, json2list
-
+gdal_warning = 'No GDAL Installation found. Without gdal the following functions are not available: from_gdal, to_gdal'
```

Comment:
Fine to define it here, but write it in capital letters. This makes it clear, that it is a global variable